charts/flyte/templates/common/secret-auth.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: flyte-propeller-auth
+  name: flyte-common-auth
   namespace: {{ template "flyte.namespace" . }}
 type: Opaque
 stringData:
-  client_secret: foobar
+  client_secret: {{ .Values.configmap.admin.admin.clientSecret }}
```

Comment:
Can you drop this change from this PR? we have this ongoing PR https://github.com/flyteorg/flyte/pull/1408